### PR TITLE
[FLINK-7100] TaskManager metrics are registered twice

### PR DESCRIPTION
**Pull request by zjureel**
Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [ ] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [ ] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

**Comment by zjureel** [comment]
@zentol @aljoscha I have tried to fix this issue, please have a look when you are free, thanks

File: flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala [hunk 1]
```diff
@@ -983,12 +981,6 @@ class TaskManager(
       libraryCacheManager = Some(new FallbackLibraryCacheManager)
     }
     
-    taskManagerMetricGroup =
```
Comment (zentol) [inline flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala]: you also have to remove accesses in `disassociateFromJobManager`

File: flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/utils/TaskExecutorMetricsInitializer.java [hunk 2]
```diff
@@ -1,257 +0,0 @@
...
-import java.lang.management.ThreadMXBean;
-import java.util.List;
-
-/**
- * Utility class ot initialize {@link TaskExecutor} specific metrics.
- */
-public class TaskExecutorMetricsInitializer {
-	private static final Logger LOG = LoggerFactory.getLogger(TaskExecutorMetricsInitializer.class);
-
-	public static void instantiateStatusMetrics(
```
Comment (zentol) [inline flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/utils/TaskExecutorMetricsInitializer.java]: The code here is a more cleaned up version of `MetricUtils`, so let's keep this one and just rename it. `MetricUtils` is a rather generic name anyway.

**Comment by zjureel** [comment]
@zentol Thank you for your review, I have fix the problems you metioned as follows:
1. rename `TaskExecutorMetricsInitializer` to `MetricUtils`
2. move `taskManagerMetricGroup.close()` from `disassociateFromJobManager` to `postStop`, I think the it's neccessary to close `taskManagerMetricGroup` manually

Please have another look when you are free, thanks

**Comment by dikei** [comment]
I think this commit broke the task manager page in the web ui for me. Can someone check it?

**Pull request closed without merging**